deps/3rdparty/libdispatch/src/queue.c

```diff
@@ -682,6 +717,13 @@ _dispatch_queue_dispose(dispatch_queue_t dq)
 	}
 #endif
 
+	if (slowpath(dq->dq_specific_q)) {
+		_dispatch_queue_dispose(dq_specific_q);
+	}
+	if (slowpath(dq->dq_specific_list)) {
+		_dispatch_queue_specific_release(dq);
```

Comment:
nit: this function should take a `dq_specific_list`, it's more of a `dispatch_queue_specific_list_release`